pkgs/tools/networking/pritunl-ssh/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation {
+  pname   = "pritunl-ssh";
+  version = "1.0.1674.4";
+
+  src = fetchFromGitHub {
+    owner = "pritunl";
+    repo = "pritunl-zero-client";
+    rev = "1.0.1674.4";
+    sha256 = "07z60lipbwm0p7s2bxcij21jid8w4nyh6xk2qq5qdm4acq4k1i88";
+  };
+
+  patchPhase = ''
```

Comment:
Can you try using `patchShebangs` instead? This should be handled semi-automatically.

Reply:
I am not sure if I am getting this wrong, or maybe it seems to not be working for me for some reason.

I've tried adding `patchShebangs ssh_client.py` to the `patchPhase`, and that seems to not be working (i.e. does not output anything during the `nix-build`). I've also tried replacing `patchPhase` with `postPatch`, containing:
```
  postPatch = ''
    patchShebangs ssh_client.py
    patchShebangs ssh_host_client.py
  '';
```

which at least outputs the following during the `nix-build`:

```
...
patching script interpreter paths in ssh_client.py
patching script interpreter paths in ssh_host_client.py
...
```

But when I check the binary in `result/bin`, it still has `#!/usr/bin/env python3` at the top.

Am I doing something wrong?

Reply:
I think you need to add python3 to build inputs.

Reply:
that worked, thank you!